src/modules/keyboardmanager/common/Helpers.cpp

```diff
@@ -193,6 +194,15 @@ namespace KeyboardManagerHelper
         keyEventArray[index].ki.dwExtraInfo = extraInfo;
     }
 
+    // Function to set the dummy key events used for remapping shortcuts, required to ensure releasing a modifier doesn't trigger another action (For example, Win->Start Menu or Alt->Menu bar)
+    void SetDummyKeyEvent(LPINPUT keyEventArray, int& index, ULONG_PTR extraInfo)
+    {
+        SetKeyEvent(keyEventArray, index, INPUT_KEYBOARD, (WORD)KeyboardManagerConstants::DUMMY_KEY, 0, extraInfo);

```

Comment:
[nit] Can we use the KEYEVENTF_KEYDOWN instead of 0 for clarity?

Reply:
@alekhyareddy28 there is no KEYEVENTF_KEYDOWN constant in the windows headers. Since the API docs have constants only for the non-zero value (https://docs.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-keybdinput) we have used it directly as 0 everywhere in code where SetKeyEvent is used.

Reply:
That's interesting. Sounds good.